pkg/cidata/cidata.TEMPLATE.d/boot/25-guestagent-base.sh

```diff
@@ -53,11 +58,29 @@ else
 	# Remove legacy systemd service
 	rm -f "${LIMA_CIDATA_HOME}/.config/systemd/user/lima-guestagent.service"
 
+	docker_args="--docker-sockets=${LIMA_CIDATA_DOCKER_PORT_MONITOR_SOCKETS}"
+	containerd_args="--containerd-sockets=${LIMA_CIDATA_CONTAINERD_PORT_MONITOR_SOCKETS}"
+	kubernetes_args="--kubernetes-configs=${LIMA_CIDATA_KUBERNETES_SERVICE_WATCHER_CONFIGS}"
+
 	if [ "${LIMA_CIDATA_VSOCK_PORT}" != "0" ]; then
-		sudo "${LIMA_CIDATA_GUEST_INSTALL_PREFIX}"/bin/lima-guestagent install-systemd --debug="${LIMA_CIDATA_DEBUG}" --vsock-port "${LIMA_CIDATA_VSOCK_PORT}"
-	elif [ "${LIMA_CIDATA_VIRTIO_PORT}" != "" ]; then
-		sudo "${LIMA_CIDATA_GUEST_INSTALL_PREFIX}"/bin/lima-guestagent install-systemd --debug="${LIMA_CIDATA_DEBUG}" --virtio-port "${LIMA_CIDATA_VIRTIO_PORT}"
+		sudo "${LIMA_CIDATA_GUEST_INSTALL_PREFIX}/bin/lima-guestagent" install-systemd \
+			--debug="${LIMA_CIDATA_DEBUG}" \
+			--vsock-port "${LIMA_CIDATA_VSOCK_PORT}" \
+			"${docker_args}" \
+			"${containerd_args}" \
+			"${kubernetes_args}"
```

Comment:
Super-nitpicking, but I would always pass the arguments in the same order. So to match the OpenRC script above:

```suggestion
			--debug="${LIMA_CIDATA_DEBUG}" \
			"${docker_args}" \
			"${containerd_args}" \
			"${kubernetes_args}" \
			--vsock-port "${LIMA_CIDATA_VSOCK_PORT}"
```